consensus/polybft/fsm_test.go

```diff
@@ -647,7 +647,7 @@ func TestFSM_VerifyStateTransactions_EndOfEpochWrongValidatorsUptimeTx(t *testin
 	uptimeCounter, err := createTestUptimeCounter(t, nil, 5).EncodeAbi()
 	require.NoError(t, err)
 
-	commitEpochTx := createStateTransactionWithData(contracts.ValidatorSetContract, uptimeCounter)
+	commitEpochTx := createStateTransactionWithData(fsm.config.ValidatorSetAddr, uptimeCounter)
```

Comment:
Do we need these changes?

Reply:
https://github.com/0xPolygon/polygon-edge/pull/1005/commits/f84bf34abe858c448eaa998b6e389c301ba58708